src/witan/app/handler.clj

```diff
@@ -150,24 +150,14 @@
                                              version :- java.lang.Long]
                                :summary "Returns a forecast of the specified id and version"
                                (forecast/forecast {:id id :version version}))
-                   (sweet/POST* "/forecasts/:id" []
-                                :summary "Creates a new version of this forecast with the specified updated"
-                                (not-implemented))
+                   (sweet/POST* "/forecasts/:id/versions" {:as request}
```

Comment:
It made more sense to me, in the REST framework: we're creating a version on a forecast, so we're posting to versions on that forecast